src/file/file.rpc.ts

```diff
@@ -1,5 +1,6 @@
 import { FileService } from './file.service';
 import { IFile } from './file.interface';
+import { IUpload } from './upload.interface';
 
 const PROTO_PATH = `${__dirname}/../../protos/file.proto`;
 const grpc = require('grpc');
```

Comment:
Line `const grpc = require('grpc');` should be **after** local imports and variable declarations.
Also should use `import` instead of `require` if possible, if so then it should be **before** local imports.

Reply:
Swap `require` with `import` and put it before other local imports.